extension/upload-button.js

```diff
@@ -0,0 +1,35 @@
+/* globals pageDetect, icons */
+
+const isMac = /Mac OS X/.test(window.navigator.oscpu);
```

Comment:
@jgierer12 Thank you for working on this! The macOS check doesn't work at the moment, though.

`window.navigator.oscpu` is undefined.

I have these working:

```js
/Mac OS X/.test(window.navigator.userAgent);
/Mac OS X/.test(window.navigator.appVersion);
/Mac/.test(window.navigator.platform);
```